packages/effects-core/src/math/shape/vector-feather-renderer.ts

```
@@ -202,36 +254,51 @@ export class VectorFeatherRenderer {
     if (this.currentBbox[2] <= 0 || this.currentBbox[3] <= 0) {
       return null;
     }
-
+    // 这里计算屏幕radius。
+    const featherRadiusScreen = this.computeScreenRadius(
+      renderer, worldMatrix, setFeatherRadius
+    );  
+    // 羽化半径至少要有1px，否则只会导致一些奇奇怪怪的锯齿（当使用1px羽化时，表现为抗锯齿效果）。
+    const refinedScreenRadius = Math.max(featherRadiusScreen, 1.0);
+    // 这里传递回去修改原始羽化
+    this.featherRadius = setFeatherRadius * refinedScreenRadius / Math.max(featherRadiusScreen, 0.0001);
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)